services/command-router-base/src/main/java/org/eclipse/hono/commandrouter/impl/kafka/KafkaBasedCommandConsumerFactoryImpl.java

```diff
@@ -123,102 +94,27 @@ public KafkaBasedCommandConsumerFactoryImpl(
                 kafkaProducerFactory, kafkaProducerConfig, tracer);
         commandHandler = new KafkaBasedMappingAndDelegatingCommandHandler(tenantClient, commandTargetMapper,
                 internalCommandSender, tracer);
-        final Map<String, String> consumerConfig = kafkaConsumerConfig.getConsumerConfig("cmd-router");
+        final Map<String, String> consumerConfig = kafkaConsumerConfig.getConsumerConfig("consumer");
```

Comment:
The string value being set above is used to build the id of the Kafka client. As per the Kafka doc, _the purpose of this client id is to be able to track the source of requests beyond just ip/port by allowing a logical application name to be included in server-side request logging._

For me it seems the value `consumer` is very generic. Wouldn't it be better if is more specific such as `cmd-router-consumer` or _cmd-consumer_?